src/p4est_extended.h

```diff
@@ -44,6 +44,7 @@
 #include <p4est_mesh.h>
 #include <p4est_iterate.h>
 #include <p4est_lnodes.h>
+#include <p4est_io.h>
```

Comment:
Would it be sensible to remove this include dependency by forward declaring `struct p4est_file_context`?